gitlab-logger.yaml

```
@@ -1,7 +1,7 @@
 package:
   name: gitlab-logger
   version: 3.0.0
-  epoch: 1
+  epoch: 5
```

sorry should have seen this, think this should be 2?
```suggestion
  epoch: 2
```